tools/validate.hpp

```diff
@@ -12,8 +12,11 @@
 namespace antler {
    struct validate_project {
       inline validate_project(CLI::App& app) {
+         path = system::fs::current_path().string();
          subcommand = app.add_subcommand("validate", "Validate a project.");
-         subcommand->add_option("-p, path", path, "This is the path to the root of the project.")->default_val(".");
+         subcommand->footer(std::string(R"(Examples:)")
+               + "\n\t" + app.get_name() +R"( validate)"

```

Comment:
```suggestion
               + "\n\t" + app.get_name() +R"( validate)");

```

Reply:
That's kind of embarrassing. Fixed in b165d05.